package-lock.json

```diff
@@ -1,4034 +1,3988 @@
+      "integrity": "sha1-t3c14xXOMPa27/D4OwQVGiJEliI=",
+      "requires": {
+        "debug": "^2.3.3",
+        "define-property": "^0.2.5",
+        "extend-shallow": "^2.0.1",
+        "posix-character-classes": "^0.1.0",
+        "regex-not": "^1.0.0",
+        "snapdragon": "^0.8.1",
+        "to-regex": "^3.0.1"
+      },
+      "dependencies": {
+        "define-property": {
+          "version": "0.2.5",
+          "resolved": "https://registry.npmjs.org/define-property/-/define-property-0.2.5.tgz",
+          "integrity": "sha1-w1se+RjsPJkPmlvFe+BKrOxcgRY=",
+          "requires": {
+            "is-descriptor": "^0.1.0"
           }
-        }
-      },
-      "default-resolution": {
-        "version": "2.0.0",
-        "resolved": "https://registry.npmjs.org/default-resolution/-/default-resolution-2.0.0.tgz",
-        "integrity": "sha1-vLgrqnKtebQmp2cy8aga1t8m1oQ="
-      },
-      "define-properties": {
-        "version": "1.1.3",
-        "resolved": "https://registry.npmjs.org/define-properties/-/define-properties-1.1.3.tgz",
-        "integrity": "sha512-3MqfYKj2lLzdMSf8ZIZE/V+Zuy+BgD6f164e8K2w7dgnpKArBDerGYpM46IYYcjnkdPNMjPk9A6VFB8+3SKlXQ==",
-        "requires": {
-          "object-keys": "^1.0.12"
-        }
-      },
-      "define-property": {
-        "version": "2.0.2",
-        "resolved": "https://registry.npmjs.org/define-property/-/define-property-2.0.2.tgz",
-        "integrity": "sha512-jwK2UV4cnPpbcG7+VRARKTZPUWowwXA8bzH5NP6ud0oeAxyYPuGZUAC7hMugpCdz4BeSZl2Dl9k66CHJ/46ZYQ==",
-        "requires": {
-          "is-descriptor": "^1.0.2",
-          "isobject": "^3.0.1"
-        },
-        "dependencies": {
-          "is-accessor-descriptor": {
-            "version": "1.0.0",
-            "resolved": "https://registry.npmjs.org/is-accessor-descriptor/-/is-accessor-descriptor-1.0.0.tgz",
-            "integrity": "sha512-m5hnHTkcVsPfqx3AKlyttIPb7J+XykHvJP2B9bZDjlhLIoEq4XoK64Vg7boZlVWYK6LUY94dYPEE7Lh0ZkZKcQ==",
-            "requires": {
-              "kind-of": "^6.0.0"
-            }
-          },
-          "is-data-descriptor": {
-            "version": "1.0.0",
-            "resolved": "https://registry.npmjs.org/is-data-descriptor/-/is-data-descriptor-1.0.0.tgz",
-            "integrity": "sha512-jbRXy1FmtAoCjQkVmIVYwuuqDFUbaOeDjmed1tOGPrsMhtJA4rD9tkgA0F1qJ3gRFRXcHYVkdeaP50Q5rE/jLQ==",
-            "requires": {
-              "kind-of": "^6.0.0"
-            }
-          },
-          "is-descriptor": {
-            "version": "1.0.2",
-            "resolved": "https://registry.npmjs.org/is-descriptor/-/is-descriptor-1.0.2.tgz",
-            "integrity": "sha512-2eis5WqQGV7peooDyLmNEPUrps9+SXX5c9pL3xEB+4e9HnGuDa7mB7kHxHw4CbqS9k1T2hOH3miL8n8WtiYVtg==",
-            "requires": {
-              "is-accessor-descriptor": "^1.0.0",
-              "is-data-descriptor": "^1.0.0",
-              "kind-of": "^6.0.2"
-            }
+        },
+        "e
```

Comment:
If we have `tar` as a primary dependency, will that module get used for the requirements of other packages? Sorry for the n00b question.

Reply:
This preview sucks: link to code comment is https://github.com/UWB-ACM/uwbacm-site/pull/15/files#r286721614